<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectsWindow</class>
 <widget class="QMainWindow" name="ProjectsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>920</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>920</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Projects</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="projectsArea">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>931</width>
      <height>561</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>929</width>
       <height>559</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="project">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>176</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>900</width>
          <height>176</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <widget class="QLabel" name="projectName_1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Project Name</string>
         </property>
        </widget>
        <widget class="QPushButton" name="projectOpenBtn_1">
         <property name="geometry">
          <rect>
           <x>810</x>
           <y>140</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="project_2">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>176</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>900</width>
          <height>176</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <widget class="QLabel" name="projectName_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Project Name</string>
         </property>
        </widget>
        <widget class="QPushButton" name="projectOpenBtn_2">
         <property name="geometry">
          <rect>
           <x>810</x>
           <y>140</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="project_3">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>176</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>900</width>
          <height>176</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <widget class="QLabel" name="projectName_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Project Name</string>
         </property>
        </widget>
        <widget class="QPushButton" name="projectOpenBtn_3">
         <property name="geometry">
          <rect>
           <x>810</x>
           <y>140</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionAccount"/>
    <addaction name="actionSetting"/>
    <addaction name="actionSign_Out"/>
   </widget>
   <addaction name="menuUser"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionSign_Out">
   <property name="text">
    <string>Sign Out</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
